### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Dilipchauhan1998**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # test_repository
 To test the lambda function's operation.
-// TEST.
```
Comment (Dilipchauhan1998) [inline README.md:3]: Removed Line #1

File: README.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
...
-//gtgj
```
Comment (Dilipchauhan1998) [inline README.md:4]: Removed Line #2

**Review by Dilipchauhan1998 — COMMENTED** [review]
Removed Lines

**Review by sammarth-kapse — APPROVED** [review]
Approving

File: README.md [hunk 3]
```diff
@@ -1,4 +1,3 @@
...
+AB
```
Comment (sammarth-kapse) [inline README.md:3]: Good.

**Pull request merged**